src/AllMovies.css

```diff
@@ -0,0 +1,6 @@
+.all-movies-container {
+  padding: 20px;
+  display: grid;
```

Comment:
Grid is a great choice for displaying your movies! If you were going to make this responsive for different screen sizes, I think grid would make it easiest.